diff --git a/components/pango_opengl/CMakeLists.txt b/components/pango_opengl/CMakeLists.txt
index 8d3bf869..f1485eee 100644
--- a/components/pango_opengl/CMakeLists.txt
+++ b/components/pango_opengl/CMakeLists.txt
@@ -38,13 +38,13 @@ else()
     if(_LINUX_)
         set(OpenGL_GL_PREFERENCE "GLVND")
     endif()
-    find_package(OpenGL REQUIRED QUIET)
-    find_package(GLEW REQUIRED QUIET)
+    find_package(OpenGL REQUIRED)
+    find_package(GLEW REQUIRED)
     target_include_directories( ${COMPONENT} PUBLIC
         $<BUILD_INTERFACE:${OPENGL_INCLUDE_DIR}>
         $<BUILD_INTERFACE:${GLEW_INCLUDE_DIR}>
     )
     target_link_libraries( ${COMPONENT} PUBLIC
-        ${GLEW_LIBRARY} ${OPENGL_LIBRARIES}
+        GLEW::glew opengl::opengl
     )
 endif()
